cmd/internal/trender.go

```diff
@@ -58,7 +58,24 @@ This will render a provided manifest instance-group
 
 		replicas := viper.GetInt("replicas")
 		if replicas < 0 {
-			return errors.Errorf("%s replicas flag is empty.", tRenderFailedMessage)
+			replicas = 1
```

Comment:
This sounds weird. Could you leave a comment for future reference why this is needed?